src/main/scala/io/kx/loanapp/domain/LoanAppEntity.scala

```diff
@@ -18,8 +18,8 @@ class LoanAppEntity(context: EventSourcedEntityContext) extends AbstractLoanAppE
   override def emptyState: LoanAppDomainState = LoanAppDomainState.defaultInstance;
 
   override def submit(currentState: LoanAppDomainState, submitCommand: api.SubmitCommand): EventSourcedEntity.Effect[Empty] =
-    currentState match {
-      case LoanAppDomainState.defaultInstance =>
+    currentState.status match {
+      case LoanAppDomainStatus.STATUS_UNKNOWN =>
```

Comment:
all pattern matching in the entities are solely based on the status, so we can simply use it. Less verbose than 

```scala
case LoanAppDomainState(_,_,_,_,status,_,_,_) if status == LoanAppDomainStatus.STATUS_IN_REVIEW
```